<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPhraseFileDialog</class>
 <widget class="QDialog" name="EditPhraseFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="close_push_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>1</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>380</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="ok_push_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_push_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="instruction_push_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>89</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
